--- lua-5.1.5/src/luaconf.h.orig	2008-02-12 01:25:08.000000000 +0900
+++ lua-5.1.5/src/luaconf.h	2023-08-20 05:47:41.266877200 +0900
@@ -11,6 +11,7 @@
 #include <limits.h>
 #include <stddef.h>
 
+#define LUA_USE_LINUX 1
 
 /*
 ** ==================================================================
@@ -29,7 +30,7 @@
 #endif
 
 
-#if !defined(LUA_ANSI) && defined(_WIN32)
+#if !defined(LUA_ANSI) && 0
 #define LUA_WIN
 #endif
 
@@ -80,7 +81,7 @@
 ** hierarchy or if you want to install your libraries in
 ** non-conventional directories.
 */
-#if defined(_WIN32)
+#if 0
 /*
 ** In Windows, any exclamation mark ('!') in the path is replaced by the
 ** path of the directory of the executable file of the current process.
@@ -110,7 +111,7 @@
 ** CHANGE it if your machine does not use "/" as the directory separator
 ** and is not Windows. (On Windows Lua automatically uses "\".)
 */
-#if defined(_WIN32)
+#if 0
 #define LUA_DIRSEP	"\\"
 #else
 #define LUA_DIRSEP	"/"
