diff --git img/logo.gif img/logo.gif
index 4c89f79..5ac6c06 100644
Binary files img/logo.gif and img/logo.gif differ
diff --git src/commands.zig src/commands.zig
index b685532..614c423 100644
--- src/commands.zig
+++ src/commands.zig
@@ -895,8 +895,7 @@ pub fn publish(allocator: Allocator, repository: ?[]const u8, pkg: ?[]const u8)
             .macos => "open",
             else => "xdg-open",
         };
-        var browser = try std.ChildProcess.init(&.{ open_program, "https://github.com/login/device" }, allocator);
-        defer browser.deinit();
+        var browser = std.ChildProcess.init(&.{ open_program, "https://github.com/login/device" }, allocator);
 
         _ = browser.spawnAndWait() catch {
             try std.io.getStdErr().writer().print("Failed to open your browser, please go to https://github.com/login/device", .{});
