diff --git .gyro/zig-clap-Hejsil-github.com-7188a9fc/pkg/clap/comptime.zig .gyro/zig-clap-Hejsil-github.com-7188a9fc/pkg/clap/comptime.zig
index b440004..651b6e9 100644
--- .gyro/zig-clap-Hejsil-github.com-7188a9fc/pkg/clap/comptime.zig
+++ .gyro/zig-clap-Hejsil-github.com-7188a9fc/pkg/clap/comptime.zig
@@ -153,7 +153,7 @@ pub fn ComptimeClap(
     };
 }
 
-test "" {
+test {
     const params = comptime &.{
         clap.parseParam("-a, --aa") catch unreachable,
         clap.parseParam("-b, --bb") catch unreachable,
diff --git src/Project.zig src/Project.zig
index f3a00f8..ef79d19 100644
--- src/Project.zig
+++ src/Project.zig
@@ -217,11 +217,11 @@ pub fn findBestMatchingPackage(self: Self, alias: []const u8) !?Package {
     else if (self.packages.get(alias)) |pkg|
         return pkg
     else {
-        std.log.err("ambiguous package selection, dependency has alias of '{s}', options are:", .{alias});
+        std.log.err("ambiguous package selection, dependency has alias of '{any}', options are:", .{alias});
 
         var it = self.packages.iterator();
         while (it.next()) |entry|
-            std.log.err("    {s}, root: {s}", .{ entry.key_ptr.*, entry.value_ptr.root });
+            std.log.err("    {any}, root: {any}", .{ entry.key_ptr.*, entry.value_ptr.root });
         return error.Explained;
     }
 }
diff --git src/commands.zig src/commands.zig
index 260d9fd..46ff3e0 100644
--- src/commands.zig
+++ src/commands.zig
@@ -606,7 +606,7 @@ fn gitDependency(
                 break;
             }
         } else {
-            std.log.err("failed to find package that matched the alias '{s}', the advertised packages are:", .{alias});
+            std.log.err("failed to find package that matched the alias '{any}', the advertised packages are:", .{alias});
             var it = project.packages.iterator();
             while (it.next()) |pkgs_entry| {
                 const pkg_root = pkgs_entry.value_ptr.root orelse utils.default_root;
diff --git src/completion.zig src/completion.zig
index 9046c43..d14094b 100644
--- src/completion.zig
+++ src/completion.zig
@@ -153,7 +153,7 @@ pub const shells = struct {
                         } else if (param.long_name == null) {
                             try writer.print("-{c}", .{param.short_name});
                         } else {
-                            try writer.print("{{-{c},--{s}}}", .{ param.short_name, param.long_name });
+                            try writer.print("{{-{c},--{s}}}", .{ param.short_name orelse "", param.long_name orelse ""});
                         }
 
                         try writer.print("\"[{}]", .{std.zig.fmtEscapes(param.description)});
