diff --git src/nvim/CMakeLists.txt src/nvim/CMakeLists.txt
index 7409a5e5c..023658085 100755
--- src/nvim/CMakeLists.txt
+++ src/nvim/CMakeLists.txt
@@ -78,7 +78,7 @@ if(ENABLE_IWYU)
 endif()
 
 if(MSVC)
-  target_compile_options(main_lib INTERFACE -W3)
+  target_compile_options(main_lib INTERFACE -W3 /source-charset:utf-8 /wd4244 /wd4267 /wd4996 /wd4566)
 
   # Disable warnings that give too many false positives.
   target_compile_options(main_lib INTERFACE -wd4311 -wd4146)
