diff --git main.c main.c
index 0082068..093eee2 100644
--- main.c
+++ main.c
@@ -6957,3 +6957,11 @@ DEFUN(cursorBottom, CURSOR_BOTTOM, "Move cursor to the bottom of the screen")
     arrangeLine(Currentbuf);
     displayBuffer(Currentbuf, B_NORMAL);
 }
+
+DEFUN(cursorLineTop, CURSORLINE_TOP, "Scroll the cursor line to the top")
+{
+    Currentbuf->topLine = lineSkip(Currentbuf, Currentbuf->topLine, Currentbuf->cursorY, FALSE);
+    arrangeLine(Currentbuf);
+    displayBuffer(Currentbuf, B_NORMAL);
+}
+
diff --git proto.h proto.h
index be9d64a..d23d1f0 100644
--- proto.h
+++ proto.h
@@ -831,3 +831,5 @@ long lrand48(void);
 #endif
 
 extern Str base64_encode(const char *src, size_t len);
+extern void cursorLineTop(void);
+
