diff --git a/main.c b/main.c
index ced6f409..a1dad263 100644
--- a/main.c
+++ b/main.c
@@ -6958,3 +6958,11 @@ DEFUN(cursorBottom, CURSOR_BOTTOM, "Move cursor to the bottom of the screen")
     arrangeLine(Currentbuf);
     displayBuffer(Currentbuf, B_NORMAL);
 }
+
+DEFUN(cursorLineTop, CURSORLINE_TOP, "Scroll the cursor line to the top")
+{
+    Currentbuf->topLine = lineSkip(Currentbuf, Currentbuf->topLine, Currentbuf->cursorY, FALSE);
+    arrangeLine(Currentbuf);
+    displayBuffer(Currentbuf, B_NORMAL);
+}
+
diff --git a/proto.h b/proto.h
index 66d497e0..44b7a947 100644
--- a/proto.h
+++ b/proto.h
@@ -832,3 +832,5 @@ long lrand48(void);
 #endif
 
 extern Str base64_encode(const unsigned char *src, size_t len);
+extern void cursorLineTop(void);
+
